server/src/repositories/courseTask.repository.ts

```diff
@@ -69,10 +77,17 @@ export class CourseTaskRepository extends AbstractRepository<CourseTask> {
       .createQueryBuilder('courseTask')
       .addSelect('COUNT(taskResult.id)', 'taskResultCount')
       .leftJoin(TaskResult, 'taskResult', '"taskResult"."courseTaskId" = "courseTask"."id"')
+      .leftJoin(TaskInterviewResult, 'tir', 'tir.courseTaskId = courseTask.id AND "tir"."studentId" = :studentId', {
+        studentId: student?.id,
+      })
       .leftJoin(TaskResult, 'tr', '"tr"."courseTaskId" = "courseTask"."id" AND "tr"."studentId" = :studentId', {
         studentId: student?.id,
       })
-      .addSelect('tr.score', 'score')
+      .leftJoin(StageInterview, 'si', '"si"."studentId" = :studentId AND "si"."courseTaskId" = "courseTask"."id"', {
+        studentId: student?.id,
```

Comment:
this optional chaining is misleading: if there is no user should we execute the query for tasks? seems no. probably better to add additional check above and not use optional chaining here.